exercises/07-automate-deployment.md

```diff
@@ -1,49 +1,50 @@
 # Automate deployment
-In a real world scenario you would only deploy your changes, once they have been tested for quality. You would execute the tests and checks as well as the deployment automatically. Therefore, we provide the [SAP Cloud SDK Continuous Delivery Toolkit](https://github.com/SAP/cloud-s4-sdk-pipeline) in addition to the libraries that you have seen before.
-
-This toolkit allows you to easily setup and configure a continuuous delivery pipeline for Jenkins. It also supports Travis CI (less feature rich) and GitHub Actions (experimental).
-
-We will use Github Actions as a lightweight solution today.
+In a real-world scenario, you would only deploy your changes, once they have been tested for quality. You would execute the tests and checks as well as the deployment automatically. We will use GitHub Actions as a lightweight solution in this tutorial.
 
 ## Review the CI / CD configuration
+
 Let's take a look at some configuration files:
-* [pipeline_config.yml](../pipeline_config.yml): Configure the steps of your pipeline.
-* [build.yml](../.github/workflows/build.yml): Arrange steps for your pipeline.
 
-If you use the SAP Cloud SDK Continuous Delivery Toolkit in a productive environment you will probably rather use a Jenkins server as of today. For this you would load the pipeline through a Jenkins file and would not need to arrange your steps in the `build.yml`.
+- [.pipeline/config.yml](../.pipeline/config.yml): Configure the steps of your pipeline.
```

Comment:
[q] @marikaner is this still relevant if we use github actions?

Reply:
I removed this section.